README.md

```diff
@@ -117,37 +122,50 @@ Add the Android platform as the target for your mobile app. Then install plug-in
 ```
 $ cd mobile
 $ cordova platform add android
+```
 
-# ensure that everything has been installed correctly
+Ensure that everything has been installed correctly:
+
+```
 $ cordova requirements
+```
+
+Install the plugins required by the application:
 
+```
 $ cordova plugin add cordova-plugin-camera
 $ cordova plugin add cordova-plugin-file-transfer
 ```
 
-### 6. Setup your Android phone
+### 6. Setup your Android device
 
-In order to run the mobile app on your phone, you will need to enable `developer options` and `web debugging`.
+In order to run the application on your Android device, you will need to be prepared to transfer the application's `.apk` file to your device (created in the next step). There are multiple ways for developers to achieve this.
 
-> Note: Please refer to documentation on your specific phone to set these options.
+Android Studio will handle the transfer for you if you tether your Android device to your computer, and enable enable both `developer options` and `web debugging`.
 
-Attach your Android phone to your computer.
+> Note: Please refer to documentation on your specific phone to set these options.
 
-Install [Android File Transfer](https://www.android.com/filetransfer/) on your computer to enable file transfer between your computer and phone.
+For Mac users, [Android File Transfer](https://www.android.com/filetransfer/) will facilitate simple file transfers between your computer and Android device.
 
 ### 7. Build and run the mobile app
 
 ```
 $ cd mobile
 $ cordova build android
+```
+
+An `.apk` file should appear at `mobile/platforms/android/build/outputs/apk/android-debug.apk`, which contains the Android application. You can manually transfer the `.apk` to your device, or if your device is tethered (as described in the previous step), then you can run:
+
+```
 $ cordova run android
```

Comment:
is `cordova run android` required for both cases? I don't think this is clear either way.

Reply:
Clarified this in a subsequent commit by breaking the paragraph up and explaining the alternatives.